src/main/java/org/candlepin/subscriptions/inventory/db/InventoryRepository.java

```diff
@@ -74,20 +74,11 @@ int activeSystemCountForOrgId(
   @Query(
       nativeQuery = true,
       value =
-          "select "
-              + "distinct h.facts->'rhsm'->>'VM_HOST_UUID' as hyp_id, "
+          "select distinct h.system_profile_facts->>'virtual_host_uuid' as hyp_id, "
               + "h_.canonical_facts->>'subscription_manager_id' as hyp_subman_id "
               + "from hosts h "
-              + "left outer join hosts h_ on h.facts->'rhsm'->>'VM_HOST_UUID' = h_.canonical_facts->>'subscription_manager_id' "
-              + "where h.facts->'rhsm'->'VM_HOST_UUID' is not null "
-              + "and h.org_id IN (:orgIds)"
-              + "union all "
-              + "select "
-              + "distinct h.facts->'satellite'->>'virtual_host_uuid' as hyp_id, "
-              + "h_.canonical_facts->>'subscription_manager_id' as hyp_subman_id "
-              + "from hosts h "
-              + "left outer join hosts h_ on h.facts->'satellite'->>'virtual_host_uuid' = h_.canonical_facts->>'subscription_manager_id' "
-              + "where h.facts->'satellite'->'virtual_host_uuid' is not null "
```

Comment:
We actually still need to consult the `'satellite'->'virtual_host_uuid'` fact, because the reporting pipeline for Satellite doesn't yet use the canonical field.

I determined this via:
* https://github.com/RedHatInsights/yuptoo doesn't have any special handling of `virtual_host_uuid`
* [the satellite plugin](https://github.com/theforeman/foreman_rh_cloud/blob/05b52e2a520912cb659c27bab7023fafb1c02945/lib/foreman_inventory_upload/generators/slice.rb#L158-L160) reports `virtual_host_uuid` as a Satellite-specific fact.

Reply:
Thanks for the pointers! I've just removed the satelite changes and updated the PR and JIRA descriptions.